src/util/expr_cast.h

```diff
@@ -29,6 +29,9 @@ Author: Nathan Phillips <Nathan.Phillips@diffblue.com>
 /// \return true if \a base is of type \a T
 template<typename T> inline bool can_cast_expr(const exprt &base);
 
+/// Similar to can_cast_expr(const exprt &base) but for typet.
```

Comment:
This comment will appear in generated docs so it is worth copying the more expressive comment above.